### ISSUE TITLE
How to register matchers?

### ISSUE THREAD

**Issue by paxcodes** [kiwicom/pytest-recording#45]
I would like to [register my own matcher](https://vcrpy.readthedocs.io/en/latest/advanced.html#register-your-own-request-matcher).

I looked at `pytest-recording`'s source and noticed vcr is in _vcr.py ; Since it starts with an underscore, I'm guessing that it's not meant to be imported outside of the package?

How do I register my own matcher with pytest-recording?

**Comment by Stranger6667** [comment]
Hi! 

At the moment it is not possible - I'll work on this feature

**Comment by Stranger6667** [comment]
I think we can declare a new pytest hook, so it will be backward-compatible. Something like this:

```
def pytest_recording_configure(vcr):
    vcr.register_matcher(...)
```

**Comment by Stranger6667** [comment]
This is released in 0.8.0. Let me know if it works for you :)

**Comment by paxcodes** [comment]
Thanks for the quick response there @Stranger6667! Yes, I was able to register my matcher. 👍🏼
